src/modules/character.rs

```diff
@@ -1,4 +1,5 @@
 use super::{Context, Module};
+use crate::config::{Config, TableExt};
```

Comment:
Is this addition needed? 🤔

Reply:
Whoa, how did that get there?

I might have fumbled a key combo in my editor, but that is definitely not needed.